src/client/components/search-component/index.jsx

```diff
@@ -0,0 +1,213 @@
+import React from 'react';
+import './index.scss';
+import { Input, Button, Form, Dropdown, Header } from 'semantic-ui-react';
+import { DatesRangeInput } from 'semantic-ui-calendar-react';
+import moment from 'moment';
+
+class SearchComponent extends React.Component {
+    constructor(props) {
+        super(props);
+        this.state = {
+            destination: '',
+            date: '',
+            rooms: 1,
+            adults: 2,
+            children: 0
+        };
+    }
+
+    handleChange(event, {name, value}) {
+        this.setState({ [name]: value });
+    }
+
+    toggleRoomSelector(){
+        document.getElementById('room-selector').classList.toggle('hidden');
+    }
+
+    showRoomSelector(){
+        document.getElementById('room-selector').classList.remove('hidden');
+    }
+
+    hideRoomSelector(){
+        document.getElementById('room-selector').classList.add('hidden');
+    }
+
+    render() {
+        const selectOptions = [
+            { text: '1', value: 1 },
+            { text: '2', value: 2 },
+            { text: '3', value: 3 },
+            { text: '4', value: 4 },
+            { text: '5', value: 5 },
+            { text: '6', value: 6 },
+            { text: '7', value: 7 },
+            { text: '8', value: 8 },
+            { text: '9', value: 9 },
+            { text: '10', value: 10 }
+        ];
+        const childrenOptions = [
+            { text: '0', value: 0 },
+            { text: '1', value: 1 },
+            { text: '2', value: 2 },
+            { text: '3', value: 3 },
+            { text: '4', value: 4 },
+            { text: '5', value: 5 },
+            { text: '6', value: 6 },
+            { text: '7', value: 7 },
+            { text: '8', value: 8 },
+            { text: '9', value: 9 },
+            { text: '10', value: 10 }
+        ];
+
+
+        return (
+            this.props.view === 'bar' ?
+            <Form className='search-bar' >
+                <div className='destination'>
```

Comment:
Since we are not using a special setup that would generate class names automatically, please use the BEM naming conversion. Read more, http://getbem.com/introduction/